zebra/rtadv.c

```diff
@@ -3069,3 +3055,13 @@ uint32_t rtadv_get_interfaces_configured_from_bgp(void)
 {
 	return interfaces_configured_for_ra_from_bgp;
 }
+
+void rtadv_init(void)
+{
+	if (CMSG_SPACE(sizeof(struct in6_pktinfo)) > RTADV_ADATA_SIZE) {
+		zlog_debug("%s: RTADV_ADATA_SIZE choosen will not work on this platform, please use a larger size",
```

Comment:
To me it sounds like a warning, not debug, what do you think?

Reply:
I choose debug because this should be found by a developer when FRR is moved to a new platform.

Reply:
:heavy_check_mark: